internal/error.go

```diff
@@ -406,9 +420,20 @@ func NewContinueAsNewError(ctx Context, wfn interface{}, args ...interface{}) *C
 	return &ContinueAsNewError{wfn: wfn, args: args, params: params}
 }
 
-// Error from error interface
+// Error from error interface.
 func (e *ApplicationError) Error() string {
-	return e.message
+	msg := e.message()
+	if e.errType != "" {
+		msg = fmt.Sprintf("%s (type: %s, retryable: %v)", msg, e.errType, !e.nonRetryable)
+	}
+	if e.cause != nil {
+		msg = fmt.Sprintf("%s: %v", msg, e.cause)
+	}
```

Comment:
This will make error message looks like native error message in case if original error was  standart go error (i.e. `errors.New("message")`).